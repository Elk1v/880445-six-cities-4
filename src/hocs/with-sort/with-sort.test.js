import * as React from "react";
import * as renderer from "react-test-renderer";
import withSort from "Hocs/with-sort/with-sort.jsx";

const MockComponent = (props) => {
  return (
    <div
      {...props}
    />
  );
};

const MockComponentWrapped = withSort(MockComponent);

it(`withSort is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      isOpened={false}
      isActive={true}
      onArrowClick={() => {}}
      onPlacesOptionClick={() => {}}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
