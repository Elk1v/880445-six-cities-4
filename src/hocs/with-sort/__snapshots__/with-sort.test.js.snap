// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withSort is rendered correctly 1`] = `
<div
  isActive={false}
  isOpened={false}
  onArrowClick={[Function]}
  onPlacesOptionClick={[Function]}
/>
`;
